programs/Makefile

```diff
@@ -56,17 +56,48 @@ LZ4_VERSION=$(LIBVER)
 MD2ROFF   = ronn
 MD2ROFF_FLAGS = --roff --warnings --manual="User Commands" --organization="lz4 $(LZ4_VERSION)"
 
```

Comment:
Can't `programs/Makefile` use `include Makefile.inc` too ?

Reply:
It should.